setup_env.sh

```diff
@@ -7,4 +7,4 @@ export ZECALE_SETUP_DIR=$ZECALE/zecale_setup
 mkdir -p $ZECALE_SETUP_DIR
 
 # Add the zecale executables in the PATH
-export PATH=$ZECALE/build/aggregator_server:$ZECALE/build/libzecale/tests:$PATH
\ No newline at end of file
+export PATH=$ZECALE/build/aggregator-server:$ZECALE/build/libzecale/tests:$PATH
```

Comment:
AFAICT, the directory name here remains as `aggregator_server`.
(Also maybe worth adding the final newline here)

Reply:
Ah good catch, thanks, this is indeed an erroneous renaming here.